### PR TITLE
Landing page text

### PR DESCRIPTION
**Pull request by jwitz**
OK having trouble with the diff on this one, but the only changes here are to the `overview.md` pages and `src/pages/index.md`. Please review all of the text in these files for any typos/ weird phrasing/ correct links.

### CODE REVIEW

File: src/pages/index.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ import AstroCard from '@site/src/components/AstroCard';
...
-  <LinkCard topIcon label="Understand Key Airflow Concepts" description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud." href="/learn/astro-python-sdk" icon="/img/airflow-logo-85x85.png" />
-  <LinkCard topIcon label="Explore Step-by-step Tutorials" description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud." href="/learn/astro-python-sdk" icon="/img/doc-icon.svg" />
+  <LinkCard topIcon label="Understand Airflow concepts" description="Learn about the fundamentals of how Airflow works and best practices for running it at scale." href="/learn/category/concepts" icon="/img/airflow-logo-85x85.png" />
+  <LinkCard topIcon label="Explore tutorials" description="Follow step-by-step guides to get Airflow up and running for any use case." href="/learn/category/tutorials" icon="/img/doc-icon.svg" />
```
Comment (TJaniF) [inline src/pages/index.md:27]:
```suggestion
  <LinkCard topIcon label="Explore tutorials" description="Follow step-by-step instructions to get Airflow up and running for any use case." href="/learn/category/tutorials" icon="/img/doc-icon.svg" />
```
I think using "guides" here might be confusing for people familiar with the old guides looking to find them here? 🤔

File: software/overview.md [hunk 2]
```diff
@@ -22,23 +22,23 @@ The following diagram shows how you can run Airflow in your private cloud using
...
-  <LinkCard truncate label="Lorem Ipsum" description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud." href="/astro/cli/install-cli" />
-  <LinkCard truncate label="Lorem Ipsum" description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud." href="/astro/cli/reference" />
+  <LinkCard truncate label="Release notes" description="A list of all changes in the latest version of Astronomer Software." href="/software/release-notes" />
+  <LinkCard truncate label="Customize image" description="Guides and considerations for customizing your Astro project and fine-tuning Airflow." href="/software/customize-image" />
+  <LinkCard truncate label="Create a project" description="Crete all of the necessary files to run Airflow locally or on Astronomer Software." href="/software/create-project" />
```
Comment (TJaniF) [inline software/overview.md:43]:
```suggestion
  <LinkCard truncate label="Create a project" description="Create all of the necessary files to run Airflow locally or on Astronomer Software." href="/software/create-project" />
```

File: astro/cli/overview.md [hunk 3]
```diff
@@ -28,20 +28,20 @@ The Astro CLI is open source and built for data practitioners everywhere. The bi
...
 </LinkCardGrid>
 
 ## Featured CLI docs
 
 <LinkCardGrid>
-  <LinkCard truncate label="Release Notes" description="Astro CLI release notes" href="/astro/cli/release-notes" />
-  <LinkCard truncate label="Install the Astro CLI" description="This is where you'll find information about installing, upgrading, and uninstalling the Astro CLI." href="/astro/cli/install-cli" />
-  <LinkCard truncate label="Astro CLI command reference" description="This document contains information about all commands and settings available in the Astro CLI, including examples and flags. To get started with the Astro CLI, see Get Started." href="/astro/cli/reference" />
+  <LinkCard truncate label="Release Notes" description="A record of all changes to the Astro CLI." href="/astro/cli/release-notes" />
+  <LinkCard truncate label="Install the Astro CLI" description="Documentation for installing the CLI on all operating systems." />
```
Comment (TJaniF) [inline astro/cli/overview.md:43]:
```suggestion
  <LinkCard truncate label="Install the Astro CLI" description="Documentation for installing the CLI on all operating systems." href="/astro/cli/install-cli" />
```

**Review by TJaniF — APPROVED** [review]
Some small comments but overall LGTM 👍

File: astro/cli/overview.md [hunk 4]
```diff
@@ -28,20 +28,20 @@ The Astro CLI is open source and built for data practitioners everywhere. The bi
...
-  <LinkCard label="Advanced commands" description="Using the CLI, you can run a local Airflow environment, apply code changes, and view logs for all Airflow components." />
-  <LinkCard label="Example pytests and DAGs" description="Examples showcase important Airflow best practices and can help your team learn quickly and identify errors in your DAGs ahead of time." />
-  <LinkCard label="Browser-based authentication" description="Easy and secure browser-based authentication for Astro and Astronomer Software." />
-  <LinkCard label="Astro Cloud UI compatable" description="A robust set of commands to match functionality in the Cloud UI, including Deployment creation and environment variable modifications." />
-  <LinkCard label="CI/CD" description="Support for Deployment API keys, which you can use to automate commands as part of CI/CD workflows." />
+  <LinkCard label="Built-in Astro project directory" description="An Astro project includes all of the files required to run Airflow, including dedicated folders for your DAGs, packages, and unit tests." />
```
[the remark this answers is not in the document]
Comment (jwitz) [reply astro/cli/overview.md:31]: While these have link card styling, I think they exist just as "feature cards". They don't link to any docs on purpose!

**Review by kwcanuck — APPROVED** [review]

**Pull request merged**